### ISSUE TITLE
Data.gov meets Banner component of USWDS

### ISSUE THREAD

**Issue by hkdctol** [GSA/data.gov#3097]
## User Story

In order for the new version of Data.gov on cloud.gov to meet USWDS, Data.gov pages should meet the Banner component.

## Acceptance Criteria

- [ ] GIVEN a Data.gov user navigates to a page on Data.gov\
  THEN the USWDS banner indicates that the user is on a US Government website
 
## Background
Information on the Banner component is on https://designsystem.digital.gov/components/banner/

## Sketch
- [ ] Review the Banner component of USWDS
- [ ] Add the script found on the reference page so that Data.gov pages are identified as official US Government site

**Comment by hkdctol** [comment]
For Data.gov main site (former WordPress) - finishing work on #3891 will take care of this. For sdg.data.gov - we need to add the banner.

**Comment by hkdctol** [comment]
Adding catalog.data.gov to the list that needs banner, but deadline is not August 2022. We can create a separate ticket for catalog later if needed.

**Comment by jbrown-xentity** [comment]
sdg Repo: https://github.com/GSA/sdg-indicators-usa

**Comment by btylerburton** [comment]
Blocked by catalog pipeline issues.
